src/dataframe/io.jl

```diff
@@ -930,6 +930,178 @@ function readtable(pathname::AbstractString;
                      normalizenames = normalizenames)
 end
 
+"""
+# `inlinetable`
```

Comment:
See https://github.com/JuliaLang/julia/pull/15136 about formatting conventions for docstrings.